src/main/java/wooteco/subway/admin/controller/restcontroller/PathController.java

```diff
@@ -0,0 +1,60 @@
+package wooteco.subway.admin.controller.restcontroller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import wooteco.subway.admin.controller.QueryString;
+import wooteco.subway.admin.dto.request.LineStationCreateRequest;
+import wooteco.subway.admin.dto.request.PathRequest;
+import wooteco.subway.admin.dto.response.LineDetailResponse;
+import wooteco.subway.admin.dto.response.PathResponse;
+import wooteco.subway.admin.dto.response.WholeSubwayResponse;
+import wooteco.subway.admin.service.PathService;
+
+@RestController
+@RequestMapping("/paths")
+public class PathController {
+    private PathService pathService;
+
+    public PathController(PathService pathService) {
+        this.pathService = pathService;
+    }
+
+    @GetMapping
+    public ResponseEntity<PathResponse> findPath(@QueryString PathRequest pathRequest) {
```

Comment:
이렇게만 해도 Spring에서 알아서 넣어줘요! 
연습 해보시려고 적용 하셨을까요?? 
```suggestion
    public ResponseEntity<PathResponse> findPath(PathRequest pathRequest) {
```

Reply:
앗 넵 맞아요!
이번 단계에서 MethodArgumentResolver를 쓰는 부분이 너무 새롭게 느껴져서
한 번 적용해보고 싶어서 학습용으로 만들어봤습니다!